saleor/settings.py

```diff
@@ -348,30 +348,17 @@ def get_host():
 
 LOGOUT_ON_PASSWORD_CHANGE = False
 
+ENABLE_SEARCH = False
```

Comment:
Value of `ENABLE_SEARCH` depends only on `ELASTICSEARCH_URL`, what is it's purpose then? You could simply check whether the `ELASTICSEARCH_URL` is set and based on that determine whether the search is enabled or not (in the context processor). If introducing such setting, I would rather expect that it'll be taken from the environment. If True, expect the URL in parse it, otherwise ignore all the search-related settings at all.

Reply:
It's part of preparation for multiple search backends, I assume there will be separate flags for postgres and elastic search backends and overall ENABLE_SEARCH value would be set if either is set.